examples/platform/silabs/display/lcd.h

```diff
@@ -52,11 +52,11 @@ class SilabsLCD
 
     typedef struct dStatus
     {
-        uint8_t nbFabric     = 0;
-        bool connected       = false;
-        char networkName[50] = { "TODO" };
-        bool advertising     = false;
-        ICDMode_e icdMode    = NotICD;
+        uint8_t nbFabric              = 0;
+        bool connected                = false;
+        char networkName[MAX_STR_LEN] = { 0 };
```

Comment:
Shouldn't you use `DeviceLayer::Internal::kMaxWiFiSSIDLength` as this is what you use elsewhere

Reply:
done.